bevm/bevm_call_byzcoin_test.go

```diff
@@ -0,0 +1,514 @@
+package bevm
+
+import (
+	"math/big"
+	"testing"
+	"time"
+
+	"go.dedis.ch/cothority/v3"
+	"go.dedis.ch/cothority/v3/byzcoin"
+	"go.dedis.ch/cothority/v3/darc"
+	"go.dedis.ch/onet/v3"
+	"go.dedis.ch/onet/v3/log"
+	"golang.org/x/xerrors"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
+	err := byzcoin.RegisterGlobalContract(myValueContractID,
```

Comment:
```suggestion
	log.ErrFatal(byzcoin.RegisterGlobalContract(myValueContractID,
```

Then you can remove the `if`-branch.